social-network/src/main/java/com/levi9/socialnetwork/schedule/EventsRemovalScheduler.java

```diff
@@ -0,0 +1,35 @@
+package com.levi9.socialnetwork.schedule;
```

Comment:
Is this the same change as in here https://github.com/mijicuros00/levi9-social-network/pull/45/files ? @Savkela